### ISSUE TITLE
Cop: For values that only expect `true`, do not allow `'true'` (with quotes)

### ISSUE THREAD

**Issue by vitorgalvao** [caskroom/rubocop-cask#34]
Namely `stage_only`, `auto_updates`, and `accessibility_access`. They work (only tested `accessibility_access`) with both `true` and `'true'`, but from a stylistic point of view, only `true` should be allowed.

**Comment by MikeMcQuaid** [comment]
This repository has been deprecated and moved to Homebrew/brew in https://github.com/Homebrew/brew/pull/5190. Consider reopening this issue there if it's still needed.
